registry-manager-rest-server/src/test/java/de/adorsys/registry/manager/resource/AspspCsvResourceTest.java

```diff
@@ -125,55 +133,138 @@ public void mergeForbidden() throws Exception {
 
     @WithMockUser(roles = {"MANAGER", "DEPLOYER"})
     @Test
-    public void validateCsv_Success() throws Exception {
-        CsvFileValidationReportBO validationReportBO = new CsvFileValidationReportBO();
-        validationReportBO.valid();
+    public void validateImportCsv_Success() throws Exception {
+        FileValidationReportBO fileValidationReportBO = new FileValidationReportBO();
+        fileValidationReportBO.valid();
+
+        CsvFileImportValidationReportBO validationReportBO = new CsvFileImportValidationReportBO(1, 1, fileValidationReportBO);
+
+        FileValidationReportTO fileValidationReportTO = new FileValidationReportTO();
+        fileValidationReportTO.setValidationResult(ValidationResultTO.VALID);
+
+        CsvFileImportValidationReportTO validationReportTO = new CsvFileImportValidationReportTO();
+        validationReportTO.setCsvFileRecordsNumber(1);
+        validationReportTO.setDbRecordsNumber(1);
+        validationReportTO.setFileValidationReport(fileValidationReportTO);
+
+        when(service.validateImportCsv(any())).thenReturn(validationReportBO);
+        when(importValidationReportConverter.toCsvFileImportValidationReportTO(validationReportBO)).thenReturn(validationReportTO);
+
+        mockMvc.perform(multipart(BASE_URI + "/validate/upload")
```

Comment:
for rest api testing we should compare the response body with serialized string

Reply:
Good catch, thanks! Fixed